scikeras/wrappers.py

```diff
@@ -35,7 +35,11 @@
 from scikeras.utils.transformers import ClassifierLabelEncoder, RegressorTargetEncoder
 
 
-_kwarg_warn = "Passing `**kwargs` to `{0}` is not fully supported by the sklearn API. Instead, set arguments at initialization (`BaseWrapper({0}__{1}={2})`) or use `set_params`"
+_kwarg_warn = """Passing `**kwargs` to `{0}` is not fully supported by the sklearn API.
```

Comment:
```suggestion
_kwarg_warn = """Passing parameters during to `{0}` is not supported by the Scikit-learn API.
```

Reply:
`Passing parameters during to`?  Is that an extra `to`? Wouldn't this be confused with `fit takes no parameters`?

Reply:
I meant "parameters" as in "estimator parameters" (e.g., `batch_size`). I've modified the suggestion.

Reply:
This suggestion is now outdated, but we have largerly incorporated it, so I am going to resolve this comment.